src/components/Input.jsx

```diff
@@ -0,0 +1,14 @@
+const Input = ({ id, value, checked, className }) => {
+  return (
+    <input
+      className={`'duration-200 h-4 w-4 cursor-pointer appearance-none rounded-full transition-all'${className}`}
```

Comment:
Don't we need space btw the classes and the `className` variable? Also, I think the single quote is redundant.

```suggestion
      className={`duration-200 h-4 w-4 cursor-pointer appearance-none rounded-full transition-all ${className}`}
```

      
  
  Note that the input is not working properly, see example output of the code.
```html
<input class="'duration-200 h-4 w-4 cursor-pointer appearance-none rounded-full transition-all' checked checked:bg-themeOne-30 checked:hover:bg-themeOne-40" id="theme-choice-one" name="theme" type="radio" value="themeOne" checked="">
```